compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -95,9 +96,9 @@ private class ComposeWindow(
 
     val canvas = document.getElementById(canvasId) as HTMLCanvasElement
 
-    private fun <T : Event> HTMLCanvasElement.addTypedEvent(type: String, handler: (event: T, skikoView: SkikoView) -> Unit) {
+    private fun <T : Event> HTMLCanvasElement.addTypedEvent(type: String, handler: (event: T, skikoView: SkikoViewExtension) -> Unit) {
         addEventListener(type, { event ->
-            layer.layer?.skikoView?.let { skikoView -> handler(event as T, skikoView) }
+            layer.layer?.skikoView?.let { skikoView -> handler(event as T, skikoView as SkikoViewExtension) }
```

Comment:
Ideally we would type cast operations here, as they're not so cheap (at least in k/js) and events will happen often.